torch/quantization/observer.py

```diff
@@ -739,8 +739,8 @@ def __init__(self, bins=2048, upsample_rate=128, dtype=torch.quint8,
                                                 reduce_range=reduce_range)
         self.bins = bins
         self.register_buffer('histogram', torch.zeros(self.bins))
-        self.register_buffer('min_val', torch.tensor([]))
-        self.register_buffer('max_val', torch.tensor([]))
+        self.register_buffer('min_val', torch.tensor(float('inf')))
+        self.register_buffer('max_val', torch.tensor(float('-inf')))
```

Comment:
would we need similar changes for PerChannel Observers too? They have the same issue as HistogramObserver